internal/services/servicebus/servicebus_subscription_resource_test.go

```diff
@@ -265,6 +280,12 @@ resource "azurerm_servicebus_subscription" "import" {
 `, r.basic(data))
 }
 
+func (r ServiceBusSubscriptionResource) duplicate(data acceptance.TestData) string {
+	return fmt.Sprint(data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger,
+		"duplicate_detection_history_time_window = \"PT1H\"",
+	)
```

Comment:
i think this should be 
```suggestion
	return fmt.Sprintf(testAccServiceBusSubscription_tfTemplate, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger,
		"duplicate_detection_history_time_window = \"PT1H\"",
	)
```